app/controllers/issue_templates_controller.rb

```diff
@@ -129,48 +126,52 @@ def set_pulldown
 
       if inherit_templates.any?
         inherit_templates.each do |x|
-          group.push([x.title, x.id, {:class => "inherited"}])
+          group.push([x.title, x.id, { class: 'inherited' }])
           if x.is_default == true
              if project_default_template.blank?
-              @default_template = x
+              default_template = x

```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)